Bucketeer2/Sources/Internal/JSON/JSON.UserEvaluationsState.swift

```diff
@@ -0,0 +1,9 @@
+import Foundation
+
+extension JSON {
+    enum UserEvaluationsState: Int {
```

Comment:
Sorry for mentioning this only now, but the `UserEvaluationsState` became deprecated. The server is always returning FULL, so you can remove this.
I will remove this value from the API response next week. I have also asked to remove it in the android SDK.

Reply:
OK! I removed `UserEvaluationState`. 🔥  https://github.com/ca-dp/bucketeer-ios-sdk/pull/82/commits/471fd704f23ca220e1f09e9726eb79dc69c82fb0

Reply:
Thank you!